<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_CloudUserPanel</class>
 <widget class="QWidget" name="DB_CloudUserPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="DB_CustomFrame_3" name="frame_loading">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>360</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_loading{
border: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="DB_CustomFrame_4" name="frame_cover">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>450</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="DB_CustomMessageBox" name="messageBox" native="true">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>540</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>281</width>
     <height>688</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(65, 167, 254, 255), stop:0.949438 rgba(92, 143, 251, 255));
	border: none;
	border-top-left-radius: 8px;
	border-bottom-left-radius: 8px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>199</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string>安道白板云课件中心</string>
    </property>
   </widget>
   <widget class="DB_CustomLabel_4" name="label_userIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_telephone">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="DB_CustomLabel_1" name="label_management">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>15</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(&quot;:/images/courseware/down-2@2x.png&quot;);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_courseware">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame_courseware{
border: none;
background: rgb(245, 245, 245, 30);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_icon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>5</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_courseware">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: white;
}
</string>
     </property>
     <property name="text">
      <string>课件列表</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>641</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="DB_CustomLabel_1" name="label_close">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>16</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(&quot;:/images/close.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>50</y>
     <width>641</width>
     <height>621</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>课件列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_headerName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;
</string>
    </property>
    <property name="text">
     <string>课件名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_headerTime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>78</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;</string>
    </property>
    <property name="text">
     <string>发布时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_HeaderSize">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: gray;</string>
    </property>
    <property name="text">
     <string>大小</string>
    </property>
   </widget>
   <widget class="DB_CustomTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>641</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="DB_CustomLabel_1" name="label_up">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>68</y>
      <width>8</width>
      <height>4</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(':/images/up-5.png');</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="DB_CustomLabel_1" name="label_down">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>8</width>
      <height>4</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(':/images/down-4.png');</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_HeaderMore">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_upload">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: none;
background: rgb(200, 200, 200, 200);
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pUpload">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>570</y>
      <width>92</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(78, 155, 252);
color: white;
border: none;
border-left: none;
border-top-right-radius: 5px;
border-bottom-right-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>直接上传</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DB_CustomLabel_1</class>
   <extends>QLabel</extends>
   <header location="global">CustomWidget/db_customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomFrame_3</class>
   <extends>QFrame</extends>
   <header location="global">CustomWidget/db_customframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DB_CustomLabel_4</class>
   <extends>QLabel</extends>
   <header location="global">CustomWidget/db_customlabel.h</header>
  </customwidget>
  <customwidget>
   <class>DB_CustomFrame_4</class>
   <extends>QFrame</extends>
   <header location="global">CustomWidget/db_customframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DB_CustomMessageBox</class>
   <extends>QWidget</extends>
   <header location="global">CustomWidget/db_custommessagebox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DB_CustomTableView</class>
   <extends>QTableView</extends>
   <header location="global">CustomWidget/db_customtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
